### DISCUSSION TITLE
Path written for backup file

### DISCUSSION THREAD

**Discussion by ishu128** [Chia-Network/chia-blockchain#6222] [Support]
Chia wallet show..  Press f for your backup file.. How to write path of backup file..

**Comment by BrandtH22** [comment]
@ishu128 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - paths for commands can either be specific `C:\some_directory\` or referential `\some_directory\`

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
